src/parsec/__init__.py

```diff
@@ -741,7 +741,7 @@ def exclude(p: Parser, exclude: Parser):
     def exclude_parser(text, index):
         res = exclude(text, index)
         if res.status:
-            return Value.failure(index, 'something other than %s' %res.value)
```

Comment:
Could we use 'something other than {}'.format(res.value) instead to keep backwards compatiblity?

Thanks!